android/src/main/java/com/rivereactnative/RiveReactNativeViewManager.kt

```diff
@@ -25,15 +27,18 @@ class RiveReactNativeViewManager : SimpleViewManager<RiveReactNativeView>() {
 
       "play" -> {
         args?.let {
-          // Don't remove the !! - some versions of Android/Kotlin/Android-Studio may return null
-          val animationName = it.getString(0)!!
-          val loopMode = it.getString(1)!!
-          val direction = it.getString(2)!!
-          val isStateMachine = it.getBoolean(3)!!
-          view.run {
-            val rnLoopMode = RNLoopMode.mapToRNLoopMode(loopMode)
-            val rnDirection = RNDirection.mapToRNDirection(direction)
-            play(animationName, rnLoopMode, rnDirection, isStateMachine)
+          try {
+            val animationName = it.getString(0) ?: ""
+            val loopMode = it.getString(1) ?: "loop"
+            val direction = it.getString(2) ?: "forwards"
```

Comment:
this might silently hide an error, we should log errors if those are not provided and not use defaults and go on as nothing has happened